### PR TITLE
Require security patched version of guzzle

### PR DESCRIPTION
**Pull request by AndrewCarterUK**
Fixes: https://httpoxy.org/

### CODE REVIEW

**Comment by akrabat** [comment]
ok to test

File: composer.json [hunk 1]
```diff
@@ -3,6 +3,7 @@
     "require" : {
         "swiftmailer/swiftmailer": "v5.0.3",
         "michelf/php-markdown": "1.4.0",
+        "guzzlehttp/guzzle": "4.2.4",
```
Comment (akrabat) [inline composer.json]: Don't we want `~4.2.4` here?

Comment (AndrewCarterUK) [inline composer.json]: Happy to change it, should I change the others too?

Comment (akrabat) [inline composer.json]: We'll fix the others separately. Someone should go through and check if there's been any security updates to those libraries too…

**Comment by akrabat** [comment]
Thanks!

**Pull request merged**